distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/ServiceManagerImpl.java

```diff
@@ -263,16 +289,36 @@ private FeaturesService getFeaturesService() throws InterruptedException {
                 }
                 Thread.sleep(1000);
             }
+
         }
 
         return featuresService;
     }
 
     // TODO - we should really make this a bundle and inject this.
     private ApplicationService getApplicationService() {
+        BundleContext bundleContext = getBundleContext();
+
+        if (bundleContext == null) {
```

Comment:
This chunk of code looks like it is duplicated above. Could put it in a separate helper method.